libflux/flux-core/src/semantic/flatbuffers/types.rs

```diff
@@ -19,6 +19,7 @@ use crate::semantic::types::{
     Record,
     Tvar,
     TvarKinds,
+    Vector,
```

Comment:
So far I've been making comments that say, "you don't actually need this in the PR", so I just wanted to say here, that this code is good and it should be here :)